packages/headless/src/test/mock-facet-search.ts

```diff
@@ -9,6 +9,7 @@ export function buildMockFacetSearch(
     options: buildMockFacetSearchRequestOptions(),
     isLoading: false,
     response: buildMockFacetSearchResponse(),
+    initialNumberOfValues: buildMockFacetSearchRequestOptions().numberOfValues,
```

Comment:
This a more minor point, but an alternative would be to set the value to `0`. The mock objects do not need to reflect the logic inside the reducer (i.e. that the `initialNumberOfValues` matches the `numberOfValues`), just provide falsy starting values for the test cases to act on.